nifi-registry-core/nifi-registry-web-ui/src/main/webapp/nf-registry.js

```diff
@@ -71,9 +71,11 @@ NfRegistry.prototype = {
      * Invalidate old tokens and route to login page
      */
     logout: function () {
+        this.nfRegistryApi.deleteToLogout().subscribe(function () {
```

Comment:
Do I need to do this subscribe? Or do I just called deleteToLogout()?

Reply:
@mcgilman @scottyaslan can you please review this and provide an answer? Thank you.